functiondefextractor/extractor.log

```diff
@@ -0,0 +1,30 @@
+2020-07-27 12:56:31,685 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_c.c
+2020-07-27 12:56:31,850 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_c.c
+2020-07-27 12:56:31,850 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_cpp_code.cpp
+2020-07-27 12:56:32,047 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_cpp_code.cpp
+2020-07-27 12:56:32,047 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_guardrails.py
+2020-07-27 12:56:32,195 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_guardrails.py
+2020-07-27 12:56:32,195 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_repo.java
+2020-07-27 12:56:32,417 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\test_repo.java
+2020-07-27 12:56:32,418 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\typescript_sample.js
+2020-07-27 12:56:32,589 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\typescript_sample.js
+2020-07-27 12:56:32,589 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\typescript_sample.ts
+2020-07-27 12:56:32,751 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\src\typescript_sample.ts
+2020-07-27 12:56:32,751 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\test\TestHelloController.java
+2020-07-27 12:56:32,915 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\test\TestHelloController.java
+2020-07-27 12:56:32,915 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\tst\test.cs
+2020-07-27 12:56:33,082 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\tst\test.cs
+2020-07-27 12:56:33,082 Extracting C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\tst\TstHelloController.java
+2020-07-27 12:56:33,249 Successfully extracted C:\Users\320074769\Downloads\My_Repos\Python_Repos\functiondefextractor\test_resource\test_repo\tst\TstHelloController.java
+2020-07-27 12:56:33,249 Extraction process took 0.038 minutes
+2020-07-27 12:56:33,251 13 vaild files has been analysed
```

Comment:
This file need not be part of repo
Consider adding .gitignore file

Reply:
added .gitignore file